### ISSUE TITLE
Slots vanished in seconds, may be API is getting exploited.

### ISSUE THREAD

**Issue by kvprasoon** [cowinapi/developer.cowin#178]
@cowinapi Slots vanished in seconds (around 12/May/2021 10:20 PM), may be API is getting exploited. This will make life of a common man in trouble.

See this report: https://twitter.com/joshimanasi11/status/1392528077549236224 and
https://twitter.com/joshimanasi11/status/1392530871085780995

**Comment by deepgandhi** [comment]
> @cowinapi Slots vanished in seconds (around 12/May/2021 10:20 PM), may be API is getting exploited. This will make life of a common man in trouble.
> 
> See this report: https://twitter.com/joshimanasi11/status/1392528077549236224 and
> https://twitter.com/joshimanasi11/status/1392530871085780995

No it is not possible, because even in protected api there is no api for captcha validation. You are not able to book appointment from protected api as of now as it throws captcha error.

**Comment by gopal-amlekar** [comment]
Yes captcha is required but still the slots are getting vanished in seconds. Every time I login, the new slots appear as fully booked or Session not available. This is irrespective of age group. Don't know if it is due to the 30-minutes cache response they have put in?

**Comment by vinfur** [comment]
Public Api are cached and are late so post login exact we can find...
Private api/ Post login api are real time

**Comment by dude83759** [comment]
> Public Api are cached and are late so post login exact we can find...
> Private api/ Post login api are real time

So, how can we get real time slot availability with public API

**Comment by kvprasoon** [comment]
related:

https://github.com/cowinapi/developer.cowin/issues/193

https://github.com/cowinapi/developer.cowin/issues/153

**Comment by Shahul11** [comment]
@kvprasoon  Hey Prasoon Nice to see you here,  I felt the same everytime slots are booked, BTW are you able to get the response for this api v2/appointment/sessions/public/findByDistrict?district_id
